be/src/vec/core/field.h

```diff
@@ -147,7 +147,12 @@ class JsonbField {
         memcpy(data, x.data, size);
     }
 
-    JsonbField(JsonbField&& x) : data(x.data), size(x.size) {
+    JsonbField(JsonbField&& x) {
+        if (data) {
```

Comment:
It's not necessary for constructor function since the object is not existed yet.